index.html

```diff
@@ -8,73 +8,76 @@
       rel="stylesheet"
       href="https://unpkg.com/boxicons@latest/css/boxicons.min.css"
     />
+
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="./css/DesktopVersion.css">
+    <link rel="stylesheet" href="./css/DesktopVersion.css" />
     <title>First setup</title>
   </head>
 
   <body>
     <header>
-      <nav>
-        <ul>
-          <li class="nameJacK">Jack</li>
-          <li class="Imgnoe"><a href="#">Portfolio</a></li>
-          <li class="Imgnoe"><a href="#">About</a></li>
-          <li class="Imgnoe"><a href="#">Contact</a></li>
-          <li><img src="img/menu_FILL0_wght400_GRAD0_opsz48.png" class="ImgCodeOn" alt=""></li>
-        </ul>
+      <nav class="navbar">
+        <a href="#" class="logo"><h2>Jack</h2></a>
+        <div class="nav-links">
+          <ul>
+            <img src="img/closeTag.svg" class="closeHumberger" alt="" />
+            <li class="Coloring"><a href="#Portfolio">Portfolio</a></li>
+            <li><a href="#About">About</a></li>
+            <li><a href="#Contact">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Congratulations on making it this far :+1: . However, there are a few issues that should be addressed before your project can be approved. 

1. Please ensure that, once the mobile menu is open, when we click on any link, the menu is closed and we are directed to the corresponding section on the page, as stated in the requirement. :ok_hand: 

#### Requirement :point_down: 
![Screenshot from 2023-05-22 18-21-01](https://github.com/tchasinga/Setup-first-mobile/assets/89228743/eeb94fc9-3029-40bd-a89e-baab0345f40d)

2. The design of your mobile menu is currently different from the design on Figma. As a requirement, please adhere to the template as closely as possible.😊

#### Your design :point_down: 
![Screenshot from 2023-05-22 18-25-00](https://github.com/tchasinga/Setup-first-mobile/assets/89228743/9f305bbc-aafd-4d61-8779-b2551d2fba7f)

#### The Template
![Screenshot from 2023-05-22 18-25-29](https://github.com/tchasinga/Setup-first-mobile/assets/89228743/012cb7e3-38a6-4b0b-a633-66c6798aa26c)


#### I would recommend the following updates to help you ensure that your design looks as close to the template as possible. :point_down: 

- For the class `.nav-links` I would suggest this :point_right: `#1a3bedad` for the background color and `100vh` for the `height`
- For `.navbar .nav-links ul li` let's replace the `margin: 25px 0;` to `margin-bottom: 25px;`
- Update `.nav-links ul` to this :point_down: 
```
.nav-links ul {
    margin-top: -420px;
    margin-left: -220px;
    display: flex;
    padding-left: 0;
    flex-direction: column;
    align-items: center;
    color: white;
}
```